hera_cal/abscal.py

```diff
@@ -1492,6 +1494,7 @@ def avg_data_across_red_bls(data, antpos, wgts=None, broadcast_wgts=True, tol=1.
     Output: (red_data, red_wgts, red_keys)
     -------
     """
+    print("Warning: This function will be deprecated in the next hera_cal release.")
```

Comment:
Let's do this properly and use [`warnings.warn`](https://docs.python.org/2/library/warnings.html#warnings.warn)